timemachine/fe/atom_mapping.py

```diff
@@ -7,64 +8,64 @@
 from timemachine.fe.topology import AtomMappingError
 
 
-class CompareDistNonterminal(rdFMCS.MCSAtomCompare):
-    """
-    Custom comparator used in the FMCS code.
-    This allows two atoms to match if:
-        1. Neither atom is a terminal atom (H, F, Cl, Halogens etc.)
-        2. They are within 1 angstrom of each other.
-    """
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-    def compare(self, p, mol1, atom1, mol2, atom2):
+def mcs(a, b, threshold: float = 2.0, timeout: int = 5, smarts: Optional[str] = None, conformer_aware=True, retry=True):
```

Comment:
Renamed `mcs_map` -> `mcs` + consolidated with `mcs_map_graph_only_complete_rings`  which can now be emulated by setting `conformer_aware=False`